packages/osmosis-std/src/types/cosmos/base/v1beta1.rs

```diff
@@ -14,7 +14,7 @@ use osmosis_std_derive::CosmwasmExt;
     ::schemars::JsonSchema,
     CosmwasmExt,
 )]
-#[proto_message(type_url = "/cosmos.base.v1beta1.CoinCoin")]
```

Comment:
these type urls seem off (both before and after)

Reply:
totally.. wondering why.. gotta find time to have a look

Reply:
Pushed a fix for this just now. Not sure if this is the right approach as I'm not familiar with how the file descriptors work, but it seems to work. Let me know what you think.

Reply:
Pushed a fix for this just now. Not sure if this is the right approach as I'm not familiar with how the file descriptors work, but it seems to work. Let me know what you think.